config-templates/ridge/ridge-csi-driver/1.0.0/custom-parameters.json

```diff
@@ -1,21 +1,76 @@
 [
   {
-    "description": "Set to 'true' if the location supports offline volume expansion.",
-    "displayname": "Offline expansion",
-    "name": "offline-expansion",
-    "default": "false",
-    "regex": "^(?:true|false)$",
-    "min-length": 4,
-    "max-length": 5,
-    "supported-values": ["true", "false"],
-    "type": "boolean",
+    "description": "Ridge CSI image (DaemonSet)",
+    "displayname": "Ridge CSI image (DaemonSet)",
+    "name": "ridge-csi-daemonset-image",
```

Comment:
I think we can use consistent and short param name for all Ridge CSI parameters.
I think we dont need to include `Ridge CSI` keyword for these parameters.

`ridge-csi-daemonset-image` --> `daemonset-image`